controllers/addresspool_controller.go

```diff
@@ -213,3 +230,58 @@ func (r *AddressPoolReconciler) SetupWithManager(mgr ctrl.Manager) error {
 		For(&starlingxv1.AddressPool{}).
 		Complete(r)
 }
+
+// Verify whether we have annotation restore-in-progress
+func (r *AddressPoolReconciler) checkRestoreInProgress(instance *starlingxv1.AddressPool) bool {
```

Comment:
nit: change it to: isRestoreInProgress

Reply:
Updated the func name to isRestoreInProgress